src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -280,38 +301,39 @@ private static final Path newArgFile(CompileOptions ops, String[] javaSources, P
   private static void removeTmp(Path tmp) throws IOException {
     if (tmp != null) {
       Files.walkFileTree(tmp, new SimpleFileVisitor<Path>() {
-         @Override
```

Comment:
side note; all the cosmetic changes are making this hard to review. I have to carefully read each line to make sure it does not contain a semantic change. Could we possibly have a minimal change with no cosmetic changes and send a second PR that has the formatting changes? This kind of separation significantly reduces the review difficulty.